README.md

```diff
@@ -25,7 +25,7 @@ npm start
 
 This will start the development server on http://localhost:8080/en/. This should reload automatically when you make changes to the code, but no code is perfect, so sometimes you may need to restart it. :)
 
-Note: You'll need Node.js v4 or newer as the build system uses some native ES2015 features.
+Note: You'll need Node.js 8.x or newer as the build system uses some native ES2015 features.
```

Comment:
We could probably just remove that line entirely?

Reply:
Agreed. I just went with the minimum changes :P